src/types.h

```diff
@@ -313,6 +314,11 @@ SizedType CreateUInt64();
 SizedType CreateString(size_t size);
 SizedType CreateArray(size_t num_elements, const SizedType &element_type);
 SizedType CreatePointer(const SizedType &pointee_type);
+/**
+   Create a record type
+
+   \param size size in bits
```

Comment:
A bit confusing that `size` is in bits for Type::record. The comment next to `size_t size` in `struct SizedType` says size is in bytes. Anything we can do about that?